paper/paper.bib

```diff
@@ -0,0 +1,184 @@
+@article{ac2000,
+  author = {A Agresti and B Caffo},
+  year = {2000},
+  title = {Simple and Effective Confidence Intervals for Proportions and Differences of Proportions Result from Adding Two Successes and Two Failures},
+  journal = {The Americal Statistician},
+  volume = {54},
+  number = {4},
+  pages = {280-288},
+  doi = {10.2307/2685779}
+}
+
+@article{barker2002,
+  author = {L Barker},
+  title = {A Comparison of Nine Confidence Intervals for a Poisson Parameter When the Expected Number of Events is ≤ 5},
+  journal = {The Americal Statistician},
+  volume = {56},
+  number = {2},
+  pages = {85-89},
+  year = {2002},
+  doi = {10.1198/000313002317572736}
+  }
+
+  @article{ba1986,
+  author = {J M Bland and D G Altman},
+  title = {Statistical methods for assessing agreement between two methods of clinical measurement },
+  year = {1986},
+  journal = {Lancet},
+  volume = {i(8476)},
+  pages = {307-310},
+  doi = {10.1016/S0140-6736(86)90837-8}
+}
+
+@ARTICLE{bland2007,
+  author = {J M Bland and D G Altman},
+  title = {Agreement between methods of measurement with multiple observations
+	per individual},
+  journal = {Journal of Biopharmaceutical Statistics},
+  year = {2007},
+  volume = {17},
+  pages = {571-582},
+  number = {4},
+  doi = {10.1080/10543400701329422},
+  url = {https://www.ncbi.nlm.nih.gov/pubmed/17613642}
+}
+
+@article{bland2009,
+  author = {Bland, John Martin},
+	title = {The tyranny of power: is there a better way to calculate sample size?},
+	volume = {339},
+	elocation-id = {b3985},
+	year = {2009},
+	doi = {10.1136/bmj.b3985},
+	publisher = {BMJ Publishing Group Ltd},
+	issn = {0959-8138},
+	URL = {https://www.bmj.com/content/339/bmj.b3985},
+	eprint = {https://www.bmj.com/content},
+	journal = {BMJ}
+}
+
+@article{bonnett2002,
+  author = {D G Bonnett},
+  title = {Sample size requirements for estimating intraclass correlations with desired precision},
+  year = {2002},
+  journal = {Statistics in Medicine},
+  volume = {21},
+  pages = {1331-1335},
+  doi = {10.1002/sim.1108}
+}
+
+@article{brown2001,
+  author = {L D Brown and T T Cai and A DasGupta},
+  title = {Interval Estimation for a Binomial Proportion},
+  year = {2001},
+  journal = {Statistical Science},
+  doi = {10.1214/ss/1009213286},
+  volume = {16},
+  number = {2},
+  pages = {101-117},
+  doi = {10.1214/ss/1009213286}
+}
+
+@article{bw2000,
+  author = {D G Bonnett and T A Wright},
+  year = 2000,
```

Comment:
Does it change anything if 2000 instead of {2000}?

Reply:
nope